app/client/jsx/Room.jsx

```diff
@@ -95,7 +96,11 @@ class Room extends Component {
         if (this.props.rooms[this.state.room].description) {
             return (
                 <div className="room-content">
-                    {this.props.rooms[this.state.room].description}
+                    <ReactMarkdown
+                        source={this.props.rooms[this.state.room].description}
+                        allowedTypes={['text', 'break', 'emphasis', 'strong', 'delete', 'link']}
```

Comment:
ignorable nit: i'd pull this list out to a constant

Reply:
(For future reference): we discussed; agreed that this would be lovely but that there's not an immediately obvious place to put this.